README.md

```diff
@@ -1,5 +1,51 @@
-# googleMaps-offline-navigator
-Open source project for Grow with Google Udacity Scholarship Challenge - Navigation app using offline first strategy and google maps api
+# Getting Started
+See the Guide on how to contribute [here](https://github.com/TheDevPath/googleMaps-offline-navigator/blob/development/CONTRIBUTING.md#how-to-contribute) for instructions on how to fork and set up your repository.
+
+# Installing Dependencies
+In the root directory of your newly cloned project `npm install`
+
+
+Skip this next part if you know what you are doing
+
+---
+
+Noob tip 
+
+*If you can, "clone with `SSH` instead of clone with `HTTPS`. This means that, when you type in git remote add origin, you should use a link that looks like this: `git@github.com:*YOUR_USER_NAME/YOUR_REPO_NAME.git.*` Observe how that differs from* `https://github.com/YOUR_USER_NAME/YOUR_REPO_NAME.git`* 
+While the first creates a remote that uses `ssh` authentication, the latter uses `https`, so it'll always prompt you to enter your username and password to authenticate the connection. For more see this [link](https://gist.github.com/juemura/899241d73cf719de7f540fc68071bd7d)*
+
+---
+
+# Get Google Maps API key
+
+- In the config subdirectory you will find secrets-*example.json*. Rename this to *secrets.json*. 
+
+- Next go to the Google API Console to get a Google Maps API key
+
+    - Create or select a project.
+
+    - Click Continue to enable the Google Maps API.
+
+    - On the Credentials page, get an API key. Note: If you have an existing API key, you may use that key.
+[Detailed instructions](https://developers.google.com/maps/documentation/android-api/signup)
```

Comment:
we will be using the js api here is the equivalent link https://developers.google.com/maps/documentation/javascript/get-api-key

Reply:
Thanks! Should I change and submit a new PR ?